src/test/resources/local-pipeline/run_local_pipeline

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+#Usage: $0 usgs_username usgs_password [landsat_5 | landsat_7 | landsat_8] region_pathrow image_date
+#For example: ./run_local_pipeline <usgs_username> <usgs_password> landsat_5 215065 2011-10-18
+
+#Checking args
+if [ $# -ne 5 ]
+then
+  exit 1
+fi
+
+#Capture args
+USGS_USERNAME=$1
+USGS_PASSWORD=$2
+IMAGE_DATASET=$3
+IMAGE_REGION=$4
+IMAGE_DATE=$5
+
+#Global variables
+#Exit code variable
+EXIT_CODE_PROCESS=
+
+#Declare name images
+IMAGE_NAME_INPUTDOWNLOADER="fogbow/inputdownloader:latest"
+IMAGE_NAME_WORKER="fogbow/worker:latest"
+
+#Declare name containers
+CONTAINER_NAME_INPUTDOWNLOADER="inputdownloader_container"
+CONTAINER_NAME_WORKER="worker_container"
+
+function check_docker {
+  which docker
+}
+
+function pull_image_inputdownloader {
+  docker pull $IMAGE_NAME_INPUTDOWNLOADER
+}
+
+function pull_image_preprocessor {
+  :
```

Comment:
instead of this, refactor this function and the other pull functions to do nothing when the string is empty

Reply:
the run function also should test if the name is empty